pandas/core/algorithms.py

```diff
@@ -436,6 +437,7 @@ def isin(comps, values):
     return f(comps, values)
 
 
+@deprecate_kwarg(old_arg_name='order', new_arg_name=None)
```

Comment:
ca you add a test which catches this, AND see if we have any existing cases of actually passing this arg

Reply:
Sure thing.  I added two additional tests.  Please refer to 699805198f7e31e3f6787b612ec507dd71c056a6.

I did not find any cases in the Repo where ``factorize()`` is called with an ``order`` parameter.